### DISCUSSION TITLE
Is version 4 good for production?

### DISCUSSION THREAD

**Discussion by ccfiel** [isar-community/isar#106] [General]
Has somebody use version 4 in production?

**Comment by FritzMatthaeus** [comment]
I was having the same question. What do the maintainers think?

**Comment by jbarasa** [comment]
Not yet ready for production. But you can try it out in your personal projects

**Reply by FritzMatthaeus** [reply]
Thank you for your feedback. I have decided to stick with v3

**Reply by MrCsabaToth** [reply]
How do you try? https://github.com/isar-community/isar/discussions/105

**Comment by illusive-ch** [comment]
Any  idea when this would be ready for  production? Trying to get WASM to work with web but need 4.0 for that to work.

**Reply by jbarasa** [reply]
The original maintainer abandoned the project but according to what I heard from the community, V4 might never be ready for production because the community version has a few maintainers and as for now, they are maintaining the v3 updating dependencies and fixing bugs.
